playground/signalr/SignalRServerlessWeb/Program.cs

```diff
@@ -1,21 +1,17 @@
 using Microsoft.Azure.SignalR.Management;
-using SignalRServerlessWeb;
 
 var builder = WebApplication.CreateBuilder(args);
 
-// Add services to the container.
-builder.Services.AddRazorPages();
-builder.Services.AddHttpClient();
-builder.Services.AddSingleton(sp =>
-{
-    return new ServiceManagerBuilder()
+var serviceManager = new ServiceManagerBuilder()
         .WithOptions(option =>
         {
             option.ConnectionString = builder.Configuration["ConnectionStrings:signalrServerless"];
         })
         .BuildServiceManager();
-});
-builder.Services.AddHostedService<BackgroundWorker>();
+var hubContext = await serviceManager.CreateHubContextAsync("myHubName", default);
```

Comment:
move to right above negotiate logic

Reply:
This needs to be declared before app build to register singleton for `PeriodicBroadcaster`, so I can't really move it above negotiate logic. I can move it right above `builder.Services.AddSingleton(hubContext).AddHostedService<PeriodicBroadcaster>()`